cobigen-cli/cli/src/test/java/com/devonfw/cobigen/cli/commandtests/GenerateCommandTest.java

```diff
@@ -286,12 +286,21 @@ public void generateFromArgsWithQuote() throws IOException {
         args[4] = "--increments";
         args[5] = "15";
 
-        commandLine.execute(args);
+        execute(args);
 
         Path rootPath = outputRootFile.toPath();
         generatedList.add(rootPath.resolve("docs").toFile());
         GenerateCommandTest.deleteGeneratedFiles(generatedList);
         openApiFile.delete();
         generatedList.clear();
     }
+
+    /**
+     * This method check the return code from picocli
+     * @param args
+     */
+    private void execute(String[] args) {
+        int exitCode = commandLine.execute(args);
+        assertTrue(exitCode == 0);
```

Comment:
please use assertThat(exitCode).isEqualTo(0) to get better error messages

Reply:
there is currently no dependency for assertThat in cli

Reply:
please then include core-test. Therefore it's implemented.